### DISCUSSION TITLE
suggestion only for color saturation adjustment (fix)

### DISCUSSION THREAD

**Discussion by akores** [mapsforge/mapsforge#1451] [Q&A] [answered]
in map.rendertheme.XmlUtils.java simple version 

        if (darkTheme) {
            color = GraphicUtils.filterColor(color, Filter.INVERT);
        }

should be removed, since **darkTheme** was used for local testing only

**Comment by devemux86** [comment, marked as answer]
Discussion is in:
- https://github.com/mapsforge/mapsforge/discussions/1450.
